src/lib.rs

```diff
@@ -567,6 +597,15 @@ impl<T> Menu<T> {
         buf: &mut ratatui::buffer::Buffer,
         dropdown_count_to_go: u16, // including current, it is not drawn yet
     ) {
+        // Compute width of all menu items
+        let child_max_width = group
+            .iter()
+            .map(|menu_item| Span::from(menu_item.name.clone()).width())
+            .max()
+            .unwrap_or(0) as u16;
+        let min_drop_down_width: u16 = child_max_width + 6;
```

Comment:
better create a constant to give it a name? Or add a simple explaination why 6?

Reply:
Done.